### PR TITLE
Gen rpc docs

### PR DESCRIPTION
**Pull request by k32**

### CODE REVIEW

File: apps/emqx_conf/src/emqx_conf_schema.erl [hunk 1]
```diff
@@ -325,8 +325,6 @@ fields("db") ->
            , default => mnesia
            , desc => """
 Select the backend for the embedded database.<br/>
-<strong>Important!</strong> This setting should be the same on all nodes in the cluster.<br/>
-<strong>Important!</strong> Changing this setting in the runtime is not allowed.<br/>
```
Comment (thalesmg) [inline apps/emqx_conf/src/emqx_conf_schema.erl:329]: Q: is this allowed now?  Won't it cause problems?

File: apps/emqx_conf/src/emqx_conf_schema.erl [hunk 2]
```diff
@@ -399,75 +397,130 @@ fields("rpc") ->
...
+           })}
+    , {"keyfile",
+       sc(file(),
+          #{ mapping => "gen_rpc.keyfile"
+           , desc => "Path to the private key file for the <code>rpc.certfile</code>."
+           })}
+    , {"cacertfile",
+       sc(file(),
+          #{ mapping => "gen_rpc.cacertfile"
+           , desc => "Path to certification authority TLS certificate file used validate "
```
Comment (thalesmg) [inline apps/emqx_conf/src/emqx_conf_schema.erl:467]:
```suggestion
           , desc => "Path to certification authority TLS certificate file used to validate "
```

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_conf/src/emqx_conf_schema.erl (hunk 1 above)
Comment (k32) [reply apps/emqx_conf/src/emqx_conf_schema.erl:329]: With post commit hook it shouldn't cause much problems in EMQ X.

File: apps/emqx_conf/src/emqx_conf_schema.erl [hunk 3]
```diff
@@ -399,75 +397,130 @@ fields("rpc") ->
...
           #{ mapping => "gen_rpc.max_batch_size"
            , default => 256
+           , desc => "The maximum number of batch messages sent in asynchronous mode. "
+                     "Note that this configuration does not work in synchronous mode."
            })}
     , {"port_discovery",
        sc(hoconsc:enum([manual, stateless]),
           #{ mapping => "gen_rpc.port_discovery"
            , default => stateless
+           , desc => "<code>manual</code>: discover ports by <code>tcp_server_port</code> and <code>tcp_client_port</code>.<br/>"
```
[the remark this answers is not in the document]
Comment (k32) [reply apps/emqx_conf/src/emqx_conf_schema.erl]: Good catch, I copied it from the existing document.

**Review by thalesmg — APPROVED** [review]

**Pull request merged**